p2p/peerManager/peerManager.go

```diff
@@ -104,9 +110,16 @@ type BasicPeerManager struct {
 	*basicConnGater.BasicConnectionGater
 	*basicConnMgr.BasicConnMgr
 
+	// Handles opening and closing streams
 	streamManager streamManager.StreamManager
-	peerDBs       map[string][]*peerdb.PeerDB
 
+	// Tracks peers in different quality buckets
+	peerDBs map[string][]*peerdb.PeerDB
```

Comment:
syncMap?

Reply:
Not needed because we don't write to the map itself (except once on startup, synchronously) we write to the PeerDBs which each have their own locks.